packages/webapp/pages/plus/success.tsx

```diff
@@ -29,8 +29,10 @@ const PlusSuccessPage = (): ReactElement => {
       return;
     }
 
-    const { campaignId } = marketingCta;
-    clearMarketingCta(campaignId);
+    if (marketingCta) {
+      const { campaignId } = marketingCta;
+      clearMarketingCta(campaignId);
+    }
```

Comment:
This fixes a client error on the success page

Reply:
We should probably fix this in main.